core/snippets/snippets/import.snippet

```diff
@@ -0,0 +1,14 @@
+name: importStatement
+phrase: import
+
+---
+
+language: javascript | typescript | javascriptreact | typescriptreact
+-
+import $0 from "$0";
```

Comment:
This was the `import from` statement in @AndreasArvidsson's code; it's now just `snip import`. I kept `snip import from` for cases where the imported symbol name differs from the library name.